tests/bls/test_hash_to_curve.py

```diff
@@ -59,35 +60,30 @@ def test_iso_map_G2(iso_x, iso_y, iso_z, g2_x, g2_y):
     assert g2_y == result_y
 
 
+# Tests taken from: https://github.com/cfrg/draft-irtf-cfrg-hash-to-curve/blob/master/draft-irtf-cfrg-hash-to-curve.md#bls12381g2_xmdsha-256_sswu_ro_
 @pytest.mark.parametrize(
-    'msg,x,y',
+    'msg,x,y,H',
```

Comment:
FYI (not strongly) we can also do
```python
@pytest.mark.parametrize('H', lambda x: sha256(x).digest())
@pytest.mark.parametrize(
    'msg,x,y',
    [
        ...
    ]
)
```

then `H` would be `lambda x: sha256(x).digest()` for all these test cases.